schema/snapcraft.yaml

```diff
@@ -187,6 +190,9 @@ properties:
                 - type: string
                   minLength: 1
                 - type: number
+          completer:
```

Comment:
please remove this, there is a specific branch adding support for this with user guidance.